src/_locales/ja/messages.json

```diff
@@ -0,0 +1,723 @@
+    },
+    "intro_not_an_adblocker_paragraph": {
+        "message": "Privacy Badgerは、3つの異なるウェブサイトで同じトラッカーが見つかるとブロックを開始します。スリーストライクでアウト!",
+        "description": "Intro page paragraph."
+    },
+    "options_domain_status_filter": {
+        "message": "ステータスで絞り込み:",
+        "description": "Label for a dropdown control on the Tracking Domains options page tab."
+    },
+    "report_text": {
+        "message": "そのエラーについて簡単に説明します。",
+        "description": ""
+    },
+    "intro_donate1": {
+        "message": "Privacy Badger を使用すると、 <a href='https://www.eff.org/' target='_blank'>EFF（電子フロンティア財団）</a> 他の何百万人ものユーザーと一緒になって、プライバシーのために闘うことになります。私たちはオンラインであなたの権利を守るために戦う非営利団体です。ご参加ありがとうございます!",
+        "description": "Part of the 'donate' section on the intro page."
+    },
+    "options_domain_search_tooltip": {
+        "message": "ドメインを除外するには、検索語の前に \"-\"を付けます。例えば、 \".co -.com\" と入力すると、.co と .co.uk は表示されますが、.com は除外されます。",
+        "description": "Tooltip for an \"information\" icon next to the domain search input on the Tracking Domains options page tab."
+    },
+    "intro_disable_button": {
+        "message": "Privacy Badgerがページを壊していると思われる場合（ビデオが再生されないなど）、[このサイトでは保護を無効にする]をクリックすると、そのサイトのPrivacy Badgerをオフにすることができます。",
+        "description": "Intro page paragraph. The quoted string ('Disable') should match the verb used for the 'popup_disable_for_site' message."
+    },
+    "intro_link_policy": {
+        "message": "プライバシーポリシー",
+        "description": "Shown at the bottom of the intro page, links to the EFF software and technology projects privacy policy."
+    },
+    "report_input_label": {
+        "message": "説明",
+        "description": ""
+    },
+    "not_yet_blocked_header": {
+        "message": "Privacy Badgerは、これらのドメインがブロックされるべきかどうか、まだ決定していません。",
+        "description": "Popup domain list header text; separates blocked from haven't-yet-seen-enough-to-block potential trackers."
+    },
+    "options_show_not_yet_blocked": {
+        "message": "Privacy Badgerがまだブロックすると決めていないドメインを表示する:",
+        "description": "Label for a checkbox on the Tracking Domains options page tab. Should match wording used in the 'not_yet_blocked_header' message."
+    },
+    "non_tracker": {
+        "message": "以下のドメインは、あなたを追跡していないように見えます。",
+        "description": "Header text; separates tracking from non-tracking domains in the popup."
+    },
+    "popup_options_button": {
+        "message": "オプション",
+        "description": ""
+    },
+    "share_button_title_facebook": {
+        "message": "Facebookで共有",
+        "description": "Text that comes up when you hover over the social sharing buttons on the intro page."
+    },
+    "report_fail": {
+        "message": "すみません。何らかの問題が発生しました。",
+        "description": ""
+    },
+    "social_tooltip_pb_has_replaced": {
+        "message": "Privacy Badger has replaced this $BUTTON$ button",
```

Comment:
Untranslated.

Reply:
I forgot to translate it. I will correct it.